src/js/edit_wordles.js

```diff
@@ -1,21 +1,25 @@
 "use strict";
 
-/* global globalOptions, inverted_wordles */
+/* global inverted_wordles */
 
-// Bind onChange events for all input fields that users will change values
-inverted_wordles.bindInputFieldEvents = function (containerElm) {
+/**
+ * Bind onChange events for all input fields that users will change values.
```

Comment:
The description is fairly vague, and also the type of container accepted is not clearly explained. I see that in one call, the container is a wordleRow, and in another call it is the entire document. How do we avoid binding handlers to the same elements multiple times?

As well as explaining what is the scope of the container argument, should explain that this impl saves data in some format to the save_question endpoint as well as its various other side effects.

Reply:
At the page load that all wordles are rendered, the container is the entire document. When a new wordle is created an appended to the wordle list, this function is called again to bind events to new input fields. None of elements will have listeners bound multiple times. 

The same reason that the function for binding delete events accept a container parameter.

I will explain this in the code too. Thanks for the suggestion.